IntegrationTests/CocoapodsIntegration/fastlane/Appfile

```diff
@@ -0,0 +1,4 @@
+app_identifier ENV["COCOAPODS_INTEGRATION_APP_IDENTIFIER"]
+
+itc_team_id ENV["ITC_TEAM_ID"]
+team_id ENV["DEVELOPER_PORTAL_TEAM_ID"]
```

Comment:
This file is not present in the other integrations

Reply:
good catch, it's not needed anymore after signing got moved to match